src/main/java/by/onliner/page/ShoppingCartPage.java

```diff
@@ -61,4 +77,62 @@ public String getEmptyCartMassage() {
         logger.info(String.format("Displayed [%s] message", emptyCartMessage.getText()));
         return emptyCartMessage.getText();
     }
+
+    /***
+     * Get the item name from shopping cart list.
+     * @return name of the item
+     */
+    @Step("Read the item name")
+    public String getItemName() {
+        Waiter.waitForVisibility(itemName);
+        logger.info(String.format("Read the item name [%s] from shopping cart", itemName.getText()));
+        return itemName.getText().trim();
+    }
+
+    /***
+     * Check the complete order button fo visibility.
+     * @return visibility of button
+     */
+    @Step("Chek [Перейти к оформлению] button for visibility")
+    public boolean isCompleteOrderButtonVisible() {
+        boolean visibility = completeOrderButton.isDisplayed();
+        if (visibility) {
+            logger.info("'Перейти к оформлению' button is visible");
+        }
+        return visibility;
+    }
+
+    /***
+     * Add one more same item in the shopping cart by [+] button
+     */
+    @Step("Add one more same item in the shopping cart by [+] button")
+    public void clickQuantityInputPlusButton() {
+        Waiter.elementToBeClickable(quantityInputPlusButton);
+        quantityInputPlusButton.click();
+        logger.info("Click '+' button");
+        Waiter.sleep(2);
```

Comment:
Why do you need sleep here? can we avoid this?

Reply:
No, due to the peculiarities of Onliner